### ISSUE TITLE
Policy for handling complaints and behaviour

### ISSUE THREAD

**Issue by tef** [computeranonymous/computer#76]
- Guidelines for organizers
- What we (i.e computer github) can do if the organizer can't handle it, or person is unwilling to talk to the organizer
- Will we delist groups?

**Comment by janepipistrelle** [comment]
This is hard for something relatively unstructured.

Some people could volunteer as mediators for each group, but that only works if everyone knows who they are & then who mediates the mediators? This usually works well though; mediators could be listed on local pages.

I would suggest something along the lines of:
*if someone/thing is bothering you talk to the person (get support with this if needed)
*if that does not work, talk to the group organiser (or mediator, if they exist)
*if this appears to be a persistent problem in the group and the organisers are unable to handle it, organisers talk to ? at computer github
*if the organisers are unwilling to acknowledge a problem then?

Personally, I feel delisting is bad because penalise a group at the expense of some individuals (even if those individuals happened to be organisers) is bad.

**Comment by antifuchs** [comment]
This has come up on the IRC channel again, in the form of "who do I turn to".

The Geek Feminism Wiki has a page at http://geekfeminism.wikia.com/wiki/Conference_anti-harassment/Duty_officer for people who are supposed to help enforce the policy. I think this one could serve as a useful guide for what work we can expect meetups to do / escalation mechanisms we can ask them to provide (and there definitely should be one).

**Comment by adacable** [comment]
for reporting things I think we should have 
- organizers who are knowlegeable about most issues(the reccomended reading list mentioned in the pronouns discussion)
- organizers who are encouraged to call things out when they see things happening.
- volunteers who are outside the clique or common group, ideally who are less/differently privileged, who are an option for people to talk to. Ideally with experience of these situations before. 

The third one should be optional but highly recommended. 

We should not recommend talking to the person who messed up in the first place directly. This is because they may intimidate or blow them off, which is not a good experience.

**Comment by janepipistrelle** [comment]
@drcable, in my previous experience of mediating, talking to the person directly seemed to solve ~80% of problems, but the dynamic was possibly quite different (all-female roller derby league; most people knew each other well; group was fairly static). I've had enough times recently of calling people on stuff and being ignored/derailed and then just giving up/feeling rubbish for even saying anything that I think I agree with you.

Agree 100% with your three points above.

**Comment by tef** [comment]
I think #111 might be a duplicate of this, and as mentioned in #110 we need a guide for handling incidents, both online and offline, especially on effectively handling the people in question causing problems, and of course, making sure complaints handled with care, and most importantly the people who have suffered are supported, and not caused further issue

**Comment by janepipistrelle** [comment]
Draft policy on incident handling–please, please tell me if there is something I have said wrong or left out–I'm very much learning.

We should aim to handle all incidents in a consistent fashion. Let the person know in plain language what they did wrong and what action is being taken to deal with the situation. It is important to make this about the behaviour/words, not the person.

After this point, if possible do not engage with the person who was causing the problem and give them time to cool off. If you do need to engage with them, offer them information without jokes or attacks, and disengage respectfully if it is a lost cause (we all want to stay sane).

Ask the person/s who suffered if they would like further support. Listen to what they say and make sure their needs are fedback to the group so we can all learn and improve.

At meetings

If someone engages in behaviour that violates the code of conduct, organisers or designated volunteers may take action. Make sure that people attending the meeting are aware of who to contact (this should be on meeting pages; perhaps people should be easily identifiable?).

Our expectation is that if someone is asked to stop a harassing or otherwise inappropriate behaviour, they will do so immediately. If not the organisers may ask them to leave.

On IRC/Twitter

We can expect to get some questions such as "can cis white straight guys come to this group without feeling uncomfortable?". Channel mods/tweeters should deal with these by referring to the 101 pages. It is understandable that people will get annoyed by such questions when they have to deal with them on a daily basis so please avoid tone policing people's responses and instead state that our purpose is not education on these issues and redirect to factual information.

Similarly nobody should feel obligated to continually carry out 101-style education–such requests should be referred to our statement of purpose and the 101 links pages.

If an incident occurs such that an individual is banned, this should be logged. If they tweet/email asking about this, reiterate what they did, post the logs and (briefly) why this was problematic.

If possible channel mods/volunteers should themselves be from marginalised groups, in order to prevent micro aggressions (http://www.dailykos.com/story/2013/09/29/1242339/-Racial-and-gendered-microaggressions-and-hypersensitivity-or-one-last-straw-after-another) which can be invisible to the more privileged and lead to tone policing ((http://geekfeminism.wikia.com/wiki/Tone_argument) etc.

**Comment by AgentAntelope** [comment]
I'd replace etc. with "and other similar problems.", and IRC/twitter with "Online" but a big :+1: for that draft.

**Comment by tef** [comment]
That's looking mighty sensible. Although we ensure it is clear  that people from marginalised groups should not be expected or required to moderate. It burns you up. I think the sentences about who should be moderating may deserve to be somewhere other than incident handling. 

i.e we have advice for starting a group, we can expand on it to running and managing a group, (which will include moderation), and as part of that we can have a specific subsection for handling incidents.

either way, it's a strong :+1: from me, that's pretty excellent.

**Comment by malpinder** [comment]
I think drop or rephrase the parentheses about staying sane, since some of us aren't in the first place.

but other than that, I agree with @AgentAntelope and @tef's comments. :thumbsup: and thank you for writing it.

**Comment by ntlk** [comment]
This is fantastic, thank you for writing this up!

**Comment by tef** [comment]
Aside: I think we can finally remove the geek feminism code of conduct template from the front page, and move it to advice for starting a group

(Once this is merged)

**Comment by tef** [comment]
Proposed changes:
- Rename advice for starting a group to running a group (i.e change link and name)
- Merge in Jane's draft into the advice
- Move the geek feminism link into the advice, add in the 9worlds code of conduct as another example to learn from.

**Comment by ashedryden** [comment]
Something you might consider adding to the first section is letting the person reporting know the methods of action you intend to take before taking them. Sometimes they have a different context with the person who has violated the code of conduct - they may worry about their physical/professional/personal safety with the person, so we want to make sure that we respond in a way that does the least harm to the reporter.

Otherwise looks good to me (after the edits mentioned in the above comments).

:shipit:

**Comment by ghost** [comment]
I'm interested I starting up a group in Birmingham, but I'm worried about making sure everything is ok. Maybe there is some way to have a Line Manager or mentor setup where local organisers can turn to for advice or brainstorming on particular situations, and an access point for attendees to go if a situation isn't dealt with well enough.

Hello, btw :)  I'm Guy!

**Comment by tef** [comment]
@kx001 We have the irc channel for right now, as for mentoring, i think that would require us knowing what we are doing. I'm always happen to listen and I can put you in contact with people who know way more than I do.

**Comment by janepipistrelle** [comment]
@ashedryden that was the bit I struggled most with trying to write, so massive thanks for that suggestion.

Okay I'm going to try and merge the changed version of this. Apologies in advance if I screw up as I'm a non-web person. :blush:

**Comment by tef** [comment]
I'm making a branch now for this, and hopefully I won't just commit to gh-pages this time.
